<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertificationWidget</class>
 <widget class="QWidget" name="CertificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#certifications {
	border: 0px;
	color: #395265;
}
QGroupBox {
	border-radius: 0;
	background-color: rgba(255, 255, 255, 200);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="MySplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QTableWidget" name="certifications">
      <property name="styleSheet">
       <string notr="true">#certifications {
	alternate-background-color: rgb(243, 246, 250);
	color: #395265;
	selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #4185b6, stop:1 #1b6aa5);
}
</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Certification</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>21</number>
       </property>
       <item>
        <widget class="QLabel" name="certificationName">
         <property name="font">
          <font>
           <family>Pathway Gothic One</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #224f7f;
</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>3</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>3</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(100, 100, 100, 255), stop:0.2 rgba(100, 100, 100, 50), stop:1 rgba(255, 255, 255, 0));
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="movies">
              <property name="styleSheet">
               <string notr="true">#movies {
	border: 1px solid rgba(0, 0, 0, 100);
	alternate-background-color: rgb(243, 246, 250);
	color: #395265;
	selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #4185b6, stop:1 #1b6aa5);
}

QHeaderView::section {
    background-color: #f9f9f9;
    color: rgb(27, 105, 165);
    border: none;
    border-left: 1px solid #f0f0f0;
    font-weight: normal;
    font-size: 12pt;
    padding-top: 4px;
    padding-bottom: 4px;
    margin-top: 1px;
    margin-bottom: 1px;
}

QHeaderView::section:first {
    border: none;
}

QHeaderView::section:checked
{
}
</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Movie</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="buttonAddMovie">
                <property name="toolTip">
                 <string>Add Movie</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../MediaElch.qrc">
                  <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonRemoveMovie">
                <property name="toolTip">
                 <string>Remove Current Movie</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../MediaElch.qrc">
                  <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Double click a certification to rename it, right click to delete. If you want to merge two certifications just give them the same name.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Please keep in mind that the changes you make here (renaming or deleting certifications) will be made for every movie.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>3</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>3</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(100, 100, 100, 255), stop:0.2 rgba(100, 100, 100, 50), stop:1 rgba(255, 255, 255, 0));
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySplitter</class>
   <extends>QSplitter</extends>
   <header>smallWidgets/MySplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MediaElch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAddMovie</sender>
   <signal>clicked()</signal>
   <receiver>CertificationWidget</receiver>
   <slot>addMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveMovie</sender>
   <signal>clicked()</signal>
   <receiver>CertificationWidget</receiver>
   <slot>removeMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addMovie()</slot>
  <slot>removeMovie()</slot>
 </slots>
</ui>
